spec/acceptance/mssql_spec.rb

```diff
@@ -0,0 +1,373 @@
+require 'acceptance_spec_helper'
+
+RSpec.describe 'MSSQL sessions and MSSQL modules' do
+  include_context 'wait_for_expect'
+
+  TESTS = {
+    mssql: {
+      target: {
+        session_module: "auxiliary/scanner/mssql/mssql_login",
+        type: 'MSSQL',
+        platforms: [:linux, :osx, :windows],
+        datastore: {
+          global: {},
+          module: {
+            username: ENV.fetch('MSSQL_USER', 'sa'),
+            password: ENV.fetch('MSSQL_PASSWORD', 'yourStrong(!)Password'),
+            rhost: ENV.fetch('MSSQL_RHOST', '127.0.0.1'),
+            rport: ENV.fetch('MSSQL_RPORT', '1433'),
+            database: 'master'
+          }
+        }
+      },
+      module_tests: [
+        {
+          name: "post/test/mssql",
+          platforms: [:linux, :osx, :windows],
+          targets: [:session],
+          skipped: false,
+        },
+        {
+          name: "auxiliary/scanner/mssql/mssql_hashdump",
```

Comment:
I think this module needs updated to support windows server 2022? Or it needs a print_status message to say that it's not doing anything

i.e. version is:

```
[5] pry(#<Msf::Modules::Auxiliary__Scanner__Mssql__Mssql_hashdump::MetasploitModule>)> version_year
=> "2022"
```

But the code handles:

https://github.com/rapid7/metasploit-framework/blob/9e670d2f522922d6ba5b165a67c7c3724ef9efc5/modules/auxiliary/scanner/mssql/mssql_hashdump.rb#L156-L174